app/mailers/user_mailer.rb

```diff
@@ -21,4 +21,18 @@ def claim_submitted_notification(user, claim)
                          amount: claim.amount.format(symbol: true, decimal_mark: ".", no_cents: false),
                          link_to_claim:)
   end
+
+  def claim_created_support_notification(claim)
+    link_to_claim = claims_school_claim_url(id: claim.id, school_id: claim.school.id)
+    user_emails = claim.school_users.pluck(:email)
+
+    user_emails.each do |email|
```

Comment:
This looping shouldn't be here. Mailers should be dumb and send to only one `to:`, however it could accept multiple `cc:` or `bcc:`.
To send a mailer to each user separately, this loop should be extracted to the previous layer `app/services/claims/submit.rb`

Reply:
> I would also suggest doing this as a job, so that the user isn't left hanging while the service sends multiple emails.

@JamieCleare2525 I think that should already be covered due to the use of `.deliver_later` rather than `.deliver_now`?

https://github.com/DFE-Digital/itt-mentor-services/blob/8a0360b6554b23af53c961121d2104c44024d47c/app/services/claims/submit.rb#L29